### PR TITLE
re-create a docker-compose file

### PR DESCRIPTION
**Pull request by sbsimo**
starting from the templates/geonode-generic/1/docker-compose.yml

Needed for development environment

Note: kept using geoserver 2.12 due to issues with 2.13 (i.e. default style for uploaded layers is of type raster, even though the layer is vector)

### CODE REVIEW

**Comment by giohappy** [comment]
Thanks @sbsimo!
@afabiani could you please verify the issue reported by Simone regarding the wrong default style on 2.13?

**Comment by giohappy** [comment]
@sbsimo I asked @afabiani confirmations regarding the issue about the default style. It's something new to us. Would you kindly open an issue on GeoNode's main repo with some details to reproduce it?

Regarding this PR I would prefer to have Geoserver 2.13, because it actually works in general and for us. 
Also the 'django' container should build from the local folder instead of using the Docker Hub image. The docker compose for Rancher uses the published image because it's meant to be used as a catalog template of pre-built images. On the other hand the docker-compose of the project itself should build the image, as [it happens for geonode-project](https://github.com/geosolutions-it/geonode-project/blob/master/docker-compose.yml#L41) (currently undergoing a refactoring and upgrade)

**Comment by afabiani** [comment]
@sbsimo can you please compare it with this one?

https://github.com/geosolutions-it/geonode-project/blob/master/docker-compose.yml

**Comment by sbsimo** [comment]
@giohappy @afabiani thanks for your comments and suggestions.

Sure, I'll have a look at the geonode-project repo, as suggested.
I'll keep you posted

**Comment by sbsimo** [comment]
Thanks @giohappy for these info.

I can see also another difference (apart from the source images which come from different Docker Hubs, geosolutionsit vs geonode), that is on the celery and rabbitmq services. Indeed these two run on two ad-hoc containers in geonode-project while are (probably) included in the django service for the geonode-generic (I'm very new to docker, therefore I may be wrong with high probability ;-) )

In conclusion if the suggestion is to go for geonode-project, I continue working from that; of course if this PR is not useful you guys can ignore, no problem

**Comment by giohappy** [comment]
@sbsimo I suggest to wait for the updates to the offical repo's geonode-project docker files. They should be ready in a week or two. The current version is stale and I wouldn't consider using it...

**Comment by sbsimo** [comment]
@giohappy fine, thanks!

**Comment by giohappy** [comment]
@sbsimo the Docker compose on https://github.com/geonode/geonode-project has been updates. It works fine in our tests but it would be great if others (like you) could give it a try.
I will close this issue.
